vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthAccountCreatedFragment.kt

```diff
@@ -42,8 +44,15 @@ class FtueAuthAccountCreatedFragment @Inject constructor(
 
     private fun setupViews() {
         views.accountCreatedSubtitle.text = getString(R.string.ftue_account_created_subtitle, activeSessionHolder.getActiveSession().myUserId)
-        views.accountCreatedPersonalize.debouncedClicks { viewModel.handle(OnboardingAction.PostViewEvent(OnboardingViewEvents.OnPersonalizeProfile)) }
+        views.accountCreatedPersonalize.debouncedClicks { viewModel.handle(OnboardingAction.PersonalizeProfile) }
         views.accountCreatedTakeMeHome.debouncedClicks { viewModel.handle(OnboardingAction.PostViewEvent(OnboardingViewEvents.OnTakeMeHome)) }
```

Comment:
yes exactly ^^^ the screen is reused for both cases with slightly different button styling

| PERSONALISE ENABLED | PERSONALISE DISABLED |
| --- | --- |
|![2022-03-10T17:07:47,476297091+00:00](https://user-images.githubusercontent.com/1848238/157717058-e04ddd83-29bf-40a3-acfd-880ce28d16b0.png)|![2022-03-10T17:07:56,788929564+00:00](https://user-images.githubusercontent.com/1848238/157717055-cc2c81a8-bc6b-4d99-ad2d-ea5878e7c435.png)

I could reuse the `PERSONALIZE PROFILE` button instead of creating a separate view, what do you think?

Reply:
(I would vote to have separate views, it's easier to apply different style in the layout, and also the action are not the same so could be more confusing if we update the text and the action depending on the state)

Reply:
Okay for me as well to have 2 different views.